app/screens/more_channels/index.js

```diff
@@ -22,7 +22,10 @@ function mapStateToProps(state, ownProps) {
     const {getChannels: requestStatus} = state.requests.channels;
     const {config, license} = state.entities.general;
     const roles = getCurrentUserRoles(state);
-    const channels = getOtherChannels(state);
```

Comment:
I replaced this function cause it was giving me a really weird behavior, happy to discuss at any time